src/m1_facilitator/models/Gateway.ts

```diff
@@ -18,14 +18,18 @@
 import BigNumber from 'bignumber.js';
 import Comparable from '../../common/observer/Comparable';
 
+/**
+ * Type of gateways.
+ */
 /**
  * Type of gateways.
  */
 export enum GatewayType {
   CONSENSUS = 'consensus',
   MOST = 'most',
   ERC20 = 'erc20',
-  NFT = 'nft'
+  NFT = 'nft',
+  ERC20Gateway = "ERC20Gateway"
```

Comment:
I think this is not needed as `ERC20` would serve the purpose.

Reply:
Removed the unnecessary type.